app/javascript/components/explore/ExploreDisplayTabs.jsx

```diff
@@ -63,6 +63,79 @@ function getHasComparisonDe(exploreInfo, exploreParams, comparison) {
   return hasComparisonDe
 }
 
+/** Handle switching to a new clustering that has annotations (i.e., facets) not in previous clustering */
+export function handleClusterSwitchForFiltering(cellFilteringSelection, newCellFaceting, setCellFilteringSelection) {
+  console.log('cellFilteringSelection', cellFilteringSelection)
```

Comment:
are these debug statements, or functional?

Reply:
Ah, debug -- removed in 1694be2.